dev/ci/ci-common.sh

```diff
@@ -20,7 +20,8 @@ else
         export CI_PULL_REQUEST="$CIRCLE_PR_NUMBER"
         export CI_BRANCH="$CIRCLE_BRANCH"
     else # assume local
-        export CI_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+        CI_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
```

Comment:
This change looks a bit arbitrary. All other cases are export var=value. Why is it split here?

Reply:
This way if git fails the command fails, otherwise export will hide the failure.